### ISSUE TITLE
`Verbatim` constructors need to be removed

### ISSUE THREAD

**Issue by niazim3** [JacquesCarette/Drasil#990]
Remaining `Verbatim` sections that need to be removed include:
- [x] `Verbatim` in `DocSection` (#1462)
   - [x] GamePhysics
   - [x] NoPCM
   - [x] SSP
   - [x] SWHS
- [x] `SSDSubVerb` (#1590)
   - [x] GamePhysics
   - [x] NoPCM
   - [x] SSP 
   - [x] SWHS
- [x] `ExistSolnVerb`
   - [x] GamePhysics (#1449)

Please also remove the fixme comment related to this issue once it is ready to be closed https://github.com/JacquesCarette/Drasil/blob/04be33232799ad3fc80d8869fa7b6f2c69fbb6e3/code/drasil-docLang/Drasil/DocumentLanguage.hs#L69.

**Comment by JacquesCarette** [comment]
Comments on the 'status' of these:
- `Verbatim` in `DocSection`: currently still used in GamePhysics, NoPCM, SWHS and SSP.
- `SSDSubVerb` used in GamePhysics, NoPCM, SSP and SWHS
- `ExistSolnVerb` used in GamePhysics

None of the other 'Verb' seem to be used, so can be removed now.

**Comment by JacquesCarette** [comment]
Most of the other 'Verb' are now gone.

**Comment by samm82** [comment]
The only place that `Verb` exists now is:

https://github.com/JacquesCarette/Drasil/blob/697d8cff67577af5cf650126de4228a5ad2c8653/code/drasil-docLang/Drasil/DocumentLanguage.hs#L94-L96

Is this desired, or should it be removed? @JacquesCarette

**Comment by JacquesCarette** [comment]
Can you show me anywhere that this `Verb` is used? If it is never used, remove it.

**Comment by samm82** [comment]
It is only used in SSP to explain the notation of slice indexing, which is specific to the example:
https://github.com/JacquesCarette/Drasil/blob/5e3356ef40baaad3da39b593c51c95fc46cdaa57/code/drasil-example/Drasil/SSP/Body.hs#L231-L234

For comparison, in SWHS, it is not used, as each part of the ToS intro is generated:
https://github.com/JacquesCarette/Drasil/blob/5e3356ef40baaad3da39b593c51c95fc46cdaa57/code/drasil-example/Drasil/SWHS/Body.hs#L199-L201

Should this be pursued and a new issue created, or is this OK? @JacquesCarette

**Comment by JacquesCarette** [comment]
As this is related to content, I think @smiths needs to be involved. But it's probably best to open a new issue with the above text in it.

**Comment by smiths** [comment]
@samm82, I don't think we should remove the text after `Verb`; it serves a purpose.  I'm completely fine if we don't use `Verb` to present it.

**Comment by JacquesCarette** [comment]
In other words, we need to understand a bit more what that text in `Verb` is saying. If it is saying something 'semantic', then we can extend `TConvention` with that.

So, looking at that myself, I think what it is saying is (currently) too specialized to be made generic, so that using `Verb` is likely the better solution.  So this one use of `Verb` can/should stay.

**Comment by samm82** [comment]
@JacquesCarette That's what I thought too, but I just wanted to make sure.
